manifests/exporters/node-exporter-svc.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
     app: node-exporter
   annotations:
     prometheus.io/scrape: 'true'
+    prometheus/job_name: node
```

Comment:
`.io` missing.

How about just `prometheus.io/job`, `job_name` is really only specific to the Prometheus scrape config whereas the result is the plain `job` label.